src/main/java/domain/Profit.java

```diff
@@ -0,0 +1,21 @@
+package domain;
+
+import domain.lottoticket.NumberOfLottoTicket;
+
+import java.util.List;
+
+public class Profit {
+    private static int totalLottoPrice;
+    public Profit(List<WinningStatus> lottoPrices) {
+        int sumLottoPrices = sumLottoPrices(lottoPrices);
+        totalLottoPrice = sumLottoPrices;
+    }
+
+    private int sumLottoPrices(List<WinningStatus> lottoPrices){
+        return lottoPrices.stream().mapToInt(WinningStatus::getWinningMoney).sum();
+    }
+
+    public float getCalculatedProfit(NumberOfLottoTicket numberOfLottoTicket) {
+        return (float) totalLottoPrice / numberOfLottoTicket.getMoney();
```

Comment:
```suggestion
        return (float)totalLottoPrice / numberOfLottoTicket.getMoney();
```

컨벤션에 의하면 캐스팅괄호 뒤에 공백이 없어야 합니다!
[8.6 좋은 예 참조](https://naver.github.io/hackday-conventions-java/#line-wrapping)